pkg/common/CanonicalIntervalSet.go

```diff
@@ -84,6 +84,13 @@ func (i *Interval) intersection(other Interval) []Interval {
 	return []Interval{{Start: maxStart, End: minEnd}}
 }
 
+// ToIPRange returns a string of the ip range of a single interval
+func (i *Interval) toIPRange() string {
+	startIP := InttoIP4(i.Start)
+	endIP := InttoIP4(i.End)
+	return rangeIPstr(startIP, endIP)
+}
+
```

Comment:
convert to a function `intervalToIPRange(i *Interval)` within `ipBlock.go`, so that it is an internal function within the `ipblock` pkg rather than a method of `Interval`.

Reply:
done, even though I think the code looked better before :-)